---
# Documentation: https://hugoblox.com/docs/managing-content/

title: Determination of the Key Resistance Gene Analogs Involved in Ascochyta rabiei Recognition in Chickpea
subtitle: ''
summary: ''
authors:
- Ziwei Zhou
- admin
- Prabhakaran Thanjavur Sambasivam
- Rebecca Ford
tags:
- Ascochyta blight
- Chickpea
- Expression profiling
- Host resistance
- Resistance gene analogues
categories: []
date: '2019-01-01'
lastmod: 2024-01-08T10:46:49+10:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2024-01-08T00:46:48.493864Z'
publication_types:
- '2'
abstract: Chickpea (Cicer arietinum L.) is an important cool season food legume, however,
  its production is severely constrained by the foliar disease Ascochyta blight caused
  by the fungus Ascochyta rabiei (syn. Phoma rabiei). Several disease management options
  have been developed to control the pathogen, including breeding for host plant resistance.
  However, the pathogen population is evolving to produce more aggressive isolates.
  For host resistance to be effective, the plant must quickly recognize the pathogen
  and instigate initial defense mechanisms, optimally at the point of contact. Given
  that the most resistant host genotypes display rapid pathogen recognition and response,
  the approach taken was to assess the type, speed and pattern of recognition via
  Resistance Gene Analogue (RGA) transcription among resistant and susceptible cultivated
  chickpea varieties. RGAs are key factors in the recognition of plant pathogens and
  the signaling of inducible defenses. In this study, a suite of RGA loci were chosen
  for further investigation from both published literature and from newly mined homologous
  sequences within the National Center for Biotechnology Information (NCBI) database.
  Following their validation in the chickpea genome, 10 target RGAs were selected
  for differential expression analysis in response to A. rabiei infection. This was
  performed in a set of four chickpea varieties including two resistant cultivars
  (ICC3996 and PBA Seamer), one moderately resistant cultivar (PBA HatTrick) and one
  susceptible cultivar (Kyabra). Gene expression at each RGA locus was assessed via
  qPCR at 2, 6, and 24 hours after A. rabiei inoculation with a previously characterized
  highly aggressive isolate. As a result, all loci were differentially transcribed
  in response to pathogen infection in at least one genotype and at least one time
  point after inoculation. Among these, the differential expression of four RGAs was
  significant and consistently increased in the most resistant genotype ICC3996 immediately
  following inoculation, when spore germination began and ahead of penetration into
  the plantâ€™s epidermal tissues. Further in silico analyses indicated that the differentially
  transcribed RGAs function through ADP-binding within the pathogen recognition pathway.
  These represent clear targets for future functional validation and potential for
  selective resistance breeding for introgression into elite cultivars.
publication: ''
doi: 10.3389/fpls.2019.00644
links:
- name: URL
  url: https://www.frontiersin.org/articles/10.3389/fpls.2019.00644/full
---
